sources/constraint-declaration-validation.asciidoc

```diff
@@ -893,17 +893,19 @@ Examples for types being subject to implicit unwrapping are `java.util.OptionalI
 @Min(1)
 private OptionalInt optionalNumber;
 
-@Size(max=10)
-private StringProperty name;
+@Negative
+private LongProperty negativeLong;
 
 @Positive
 private IntegerProperty positiveInt;
+
+private final ListProperty<@NotBlank StringProperty> notBlankStrings;
 ----
 ====
 
-Here the `@Min`, `@Size` and `@Positive` constraints don't apply to the annotated `OptionalInt`, `StringProperty` and `IntegerProperty` themselves, but rather to the wrapped string and integer values, respectively.
+[tck-testable]#Here the `@Min`, `@Negative`, `@Positive` and `@NotBlank` constraints don't apply to the annotated `OptionalInt`, `LongProperty`, `IntegerProperty` and `StringProperty` objects themselves, but rather to the wrapped numeric and string values, respectively.#
```

Comment:
I don't think it's a good idea to add an assertion on the examples. Especially, considering the sentence below.

This is already tested with other assertions.